packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js

```diff
@@ -301,3 +303,52 @@ WithL1.story = {
     },
   },
 };
+
+export const WithFadeAnimations = ({ parameters }) => (
+  <Default parameters={parameters} />
+);
+
+WithFadeAnimations.story = {
+  name: 'With Fade Animations',
+
+  parameters: {
+    ...readme.parameters,
+    'carbon-theme': { disabled: true },
+    knobs: {
+      escapeHTML: false,
+      DotcomShell: () => {
+        const {
+          Masthead: mastheadKnobs,
+        } = mastheadStory.story.parameters.knobs;
+        const { Footer: footerKnobs } = footerStory.story.parameters.knobs;
+
+        scrollIntoView('.bx--content-block');
+
+        const delayOptions = {
+          Default: '400ms',
+          1: '1s',
+          3: '3s',
+          5: '5s',
+        };
+        const defaultDelay = '400s';
+        let delay = select('Delay (in seconds)', delayOptions, defaultDelay);
+
+        document.querySelectorAll('.bx--content-block').forEach(e => {
+          e.style.setProperty('--dds--scroll-into-view-delay', delay);
+        });
```

Comment:
I think we should better implement this in story-specific Sass file. The API confusion from @photodow may have come from this.

Reply:
No, this is not where the confusion is stemming from, but If you expect users to set this in the `SCSS` then I would mimic that behavior. "Lead by example" lol. The problem with that, that I imagine @IgnacioBecerra was going for is he wanted to set the delay programmatically via knobs.

Reply:
@photodow Yeah I can imagine that. This can be achieved by defining CSS rules like below and set the class name via knob:

```css
.dds-devenv--scroll-into-view--delay-default {
  --dds--scroll-into-view-delay: 400ms;
}

.dds-devenv--scroll-into-view--delay-slight-slow {
  --dds--scroll-into-view-delay: 1s;
}

.dds-devenv--scroll-into-view--delay-slow {
  --dds--scroll-into-view-delay: 3s;
}

.dds-devenv--scroll-into-view--delay-very-slow {
  --dds--scroll-into-view-delay: 5s;
}
```